app/controllers/api/databases.php

```diff
@@ -1802,6 +1802,8 @@ function createAttribute(string $databaseId, string $collectionId, Document $att
     ->label('audits.resource', 'database/{request.databaseId}/collection/{request.collectionId}')
     ->label('usage.metric', 'documents.{scope}.requests.create')
     ->label('usage.params', ['databaseId:{request.databaseId}', 'collectionId:{request.collectionId}'])
+    ->label('abuse-limit', 120)
```

Comment:
Why is this different than other ones?

Reply:
@Meldiron Raised a good point that some workflows require creating multiple documents at once and we don't want to be too limiting to client apps, especially when the change is already reducing the amount of requests they can make.

Reply:
@abnegate, @Meldiron, since the [abuse key is the URL](https://github.com/appwrite/appwrite/blob/a9cfb06ee92f701877642b7880d513bbe28332fb/app/controllers/shared/api.php#L48), this would allow creating 120 documents in 1 collection in 1 minute, right? Is that reasonable?

Reply:
Also, after reviewing the abuse check a bit more, it looks like [params are part of the abuse check](https://github.com/appwrite/appwrite/blob/a9cfb06ee92f701877642b7880d513bbe28332fb/app/controllers/shared/api.php#L72) so wouldn't a bad actor be able to change the param value to bypass the abuse check?

Reply:
@stnguyen90 What is reasonable is open for discussion - what are your thoughts?

We can configure the key to be whatever we want: URL, IP, any of the params, or a combination of all.

The block you referenced lets us explicitly set request params as a part of the key - if we don't use them, it won't have any effect. For example, the abuse key for creating a magic URL contains the `userId`, but none of the other params:

```php
    ->label('abuse-key', 'url:{url},userId:{param-userId}')
```

Reply:
@abnegate, 

> What is reasonable is open for discussion - what are your thoughts?

1 doc in a particular collection per second should be plenty, right?

> The block you referenced lets us explicitly set request params as a part of the key

I see now! Thanks for the clarification!

Reply:
@stnguyen90 Please check the refactor here and lets follow up there: https://github.com/appwrite/appwrite/pull/3741

> 1 doc in a particular collection per second should be plenty, right?

From further investigation today, URL keys are not populated with ID's, however, the route method was not taken into account which led to limits applying across PATCH/PUT/DELETE routes with the same URL. Following the refactor, keys will look like this:

<img width="975" alt="Screen Shot 2022-08-31 at 4 18 08 PM" src="https://user-images.githubusercontent.com/5857008/187591936-27bd51b1-7988-4f60-9acf-d655a2b6d3df.png">

cc @eldadfux